### ISSUE TITLE
Redstone Integrator does not cause blockupdates when enabling/disabling.

### ISSUE THREAD

**Issue by fatboychummy** [Seniorendi/AdvancedPeripherals#151]
### Descripe

The redstone integrator does not cause a block update (tile update?) when you enable or disable a side. This means that it will not activate things like pistons.

https://user-images.githubusercontent.com/39929480/126278053-e2f64b23-0cfa-4696-a95c-a4419245e5e8.mp4

(I would upload the above to the "Screenshots or Videos" section, but that textbox does not allow me to upload files to it)

### Steps to reproduce

1. Place a piston in front of a redstone integrator.
2. Place computer beside integrator
3. Enable the integrator

Piston does not extend until you place a block beside it (or cause some other type of update)

### Version

0.6.7b (Latest)

### Minecraft, Forge and maybe other related mods versions

Minecraft 1.16.5, Forge 36.1.32

**Comment by SirEndii** [comment]
Fixed in 0.7a

**Comment by RealGrep** [comment]
I'm not sure this is fixed. I was having issues with needing to toggle the left side redstone to get the back side to change. I just have redstone dust at the outputs to see. I messed with it a bit, and found that dropping a block at the right places caused a block update and made the redstone on the output reflect the correct state. So it seemed like a block update issue, and I searched for that and ended up here.

Thing is, I'm on Enigmatica 6 and I have version 0.7a. It's supposed to be fixed there. And I don't see any newer version out, yet.

The initial behaviour of needing the left to toggle and change state to make the back one reflect the state properly makes me wonder if it wasn't only fixed for one side, or otherwise isn't handling all the outputs on all the faces?

Anyway, in short, I don't think this is entirely properly fixed, yet. :)

Thanks!
